src/components/consul/selectors/ImageSelector.js

```diff
@@ -175,6 +226,22 @@ const styles = StyleSheet.create({
     height: imageHeight(imageWidth() * 0.6),
     width: imageWidth() * 0.6
   },
+  sueDeleteImageButton: {
+    alignItems: 'center',
+    backgroundColor: colors.accent,
```

Comment:
please take some grey, `placeholder` for example

Reply:
The reason why I prefer this color code is that in the color codes prepared for sue, `colors.accent` is the color code we want. I will update this for SVA now, but it will need to be fixed when releasing for Sue.